config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
```

Comment:
It's great there are two separate routes for /graphql and /graphiql, better than toggling back and forth!